integration/src/integration/java/pl/allegro/tech/hermes/integration/env/HermesIntegrationEnvironment.java

```diff
@@ -40,32 +60,55 @@
         STARTERS.put(WireMockStarter.class, new WireMockStarter(HTTP_ENDPOINT_PORT));
         STARTERS.put(GraphiteHttpMockStarter.class, new GraphiteHttpMockStarter());
         STARTERS.put(OAuthServerMockStarter.class, new OAuthServerMockStarter());
-        STARTERS.put(CustomKafkaStarter.class, new CustomKafkaStarter(SECONDARY_KAFKA_PORT, SECONDARY_ZK_KAFKA_CONNECT));
         STARTERS.put(JmsStarter.class, new JmsStarter());
-        STARTERS.put(ConfluentSchemaRegistryStarter.class, new ConfluentSchemaRegistryStarter(SCHEMA_REPO_PORT, SECONDARY_ZK_KAFKA_CONNECT));
-        STARTERS.put(ConsumersStarter.class, new ConsumersStarter());
-        STARTERS.put(FrontendStarter.class, new FrontendStarter(FRONTEND_PORT));
-        STARTERS.put(ManagementStarter.class, new ManagementStarter(MANAGEMENT_PORT, "integration"));
     }
 
     @BeforeSuite
     public void prepareEnvironment(ITestContext context) throws Exception {
         try {
-            for (ITestNGMethod method : context.getAllTestMethods()) {
-                method.setRetryAnalyzer(new Retry());
-            }
+            Stream.of(kafkaClusterOne, kafkaClusterTwo, hermesZookeeperOne, hermesZookeeperTwo)
+                    .parallel()
+                    .forEach(Startable::start);
+
+            schemaRegistry.start();
 
             for (Starter<?> starter : STARTERS.values()) {
                 starter.start();
             }
 
-            this.zookeeper = startZookeeperClient();
+            managementStarter = HermesManagementInstance.starter()
+                    .port(MANAGEMENT_PORT)
+                    .addKafkaCluster(DC1, kafkaClusterOne.getBootstrapServersForExternalClients())
+                    .addZookeeperCluster(DC1, hermesZookeeperOne.getConnectionString())
+                    .schemaRegistry(schemaRegistry.getUrl())
+                    .replicationFactor(kafkaClusterOne.getAllBrokers().size())
+                    .uncleanLeaderElectionEnabled(false)
+                    .start();
+
+            zookeeper = startZookeeperClient();
+
+            ConsumersStarter consumersStarter = new ConsumersStarter();
+            consumersStarter.overrideProperty(Configs.KAFKA_AUTHORIZATION_ENABLED, false);
+            consumersStarter.overrideProperty(Configs.KAFKA_CLUSTER_NAME, PRIMARY_KAFKA_CLUSTER_NAME);
+            consumersStarter.overrideProperty(Configs.KAFKA_BROKER_LIST, kafkaClusterOne.getBootstrapServersForExternalClients());
+            consumersStarter.overrideProperty(Configs.ZOOKEEPER_CONNECT_STRING, hermesZookeeperOne.getConnectionString());
+            consumersStarter.overrideProperty(Configs.SCHEMA_REPOSITORY_SERVER_URL, schemaRegistry.getUrl());
+            consumersStarter.start();
+
+            FrontendStarter frontendStarter = new FrontendStarter(FRONTEND_PORT);
+            frontendStarter.overrideProperty(Configs.KAFKA_AUTHORIZATION_ENABLED, false);
```

Comment:
Frontend doesn't need `frontendStarter.overrideProperty(Configs.KAFKA_CLUSTER_NAME, PRIMARY_KAFKA_CLUSTER_NAME);`?

Reply:
Nope, it is used only in consumers: 
![image](https://user-images.githubusercontent.com/6481553/145546322-6b92b440-cb04-403b-91e3-99324a5c74a7.png)